pkg/logs/build.go

```diff
@@ -67,7 +66,7 @@ func (bl *buildLogger) Close() error {
 }
 
 func (bl *buildLogger) Cleanup() error {
-	projectLogsDir := filepath.Join(bl.logsDir, bl.projectName, bl.buildId)
+	projectLogsDir := filepath.Join(bl.logsDir, "builds", bl.buildId)
```

Comment:
Since `projectName` is renamed, is this still a `projectLogsDir` or should it be `buildLogsDir`?